firebase-functions/functions/src/firebase/calculateResult.ts

```diff
@@ -14,7 +14,16 @@ export const calculateResult = functions.firestore.document('/players/{documentI
     try {
       const roomId = await getCurrentRoomId(context.params.documentId as string);
       const initiallyActivePlayers = await getActivePlayersInRoom(roomId);
-      if (!hasEveryoneChosen(initiallyActivePlayers)) {
+      const numberOfPlayersInTheRoom = await getNumberOfPlayers(roomId);
+      const isThereOnlyOnePlayerInTheRoom = numberOfPlayersInTheRoom < 2;
+      if (isThereOnlyOnePlayerInTheRoom)
```

Comment:
I din't think this is meant to be here :)

Reply:
Yes, testing last minute :(